explorer/src/components/Staking/OperatorsList.tsx

```diff
@@ -425,7 +425,7 @@ export const OperatorsList: FC = () => {
             <SortedTable
               data={operatorsConnection}
               columns={columns}
-              showNavigation={true}
+              showNavigation={!useRpcData}
```

Comment:
Why do this?

Reply:
Because atm when you click on the next page or change the sorting, we manipulate the graphql query and this is what change the content of the table.

When using rpc data, all the data is presented at ounce, we have not yet integrated any sorting.

And it will likely never be needed as ounce we fixed the indexing, we will likely merge both to use live data when needed, on top of indexed data